src/pages/SignIn.js

```diff
@@ -96,6 +96,25 @@ export default class SignIn extends Component {
             }, ['https://www.federalregister.gov/documents/2018/03/14/2018-05176/privacy-act-of-1974-system-of-records'])
           ])
         ]),
+        isAnvil() && div({ style: { lineHeight: 1.5, marginTop: '2rem', paddingTop: '1rem', borderTop: Style.standardLine } }, [
```

Comment:
Can we pull the style out into a shared style object, since they are identical? 

https://github.com/DataBiosphere/terra-ui/blob/dev/src/pages/workspaces/workspace/Data.js#L34 is an example of how to do that.